flexmeasures/api/v3_0/assets.py

```diff
@@ -326,3 +346,173 @@ def get_chart_data(self, id: int, asset: GenericAsset, **kwargs):
         """
         sensors = flatten_unique(asset.sensors_to_show)
         return asset.search_beliefs(sensors=sensors, as_json=True, **kwargs)
+
+    @route("/<id>/schedules/trigger", methods=["POST"])
+    @use_args(AssetTriggerSchema(), location="args_and_json", as_kwargs=True)
+    # Simplification of checking for create-children access on each of the flexible sensors,
+    # which assumes each of the flexible sensors belongs to the given asset.
+    @permission_required_for_context("create-children", ctx_arg_name="asset")
+    def trigger_schedule(
+        self,
+        asset: GenericAsset,
+        start_of_schedule: datetime,
+        duration: timedelta,
+        belief_time: datetime | None = None,
+        flex_model: dict | None = None,
+        flex_context: dict | None = None,
+        **kwargs,
+    ):
+        """
+        Trigger FlexMeasures to create a schedule for a collection of flexible devices.
```

Comment:
The trigger can also deal with inflexible devices so this is incomplete.

Reply:
The first part is true. However, no schedules are created for the inflexible devices.